config/confighttp/compressor_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
does it make sense to remove this test since its only benchmarking underlying components and not options in the collector itself?

Reply:
The only reason I think we may want to keep this, is because it essentially benchmarks the `configcompression.Type` option used in the collector (zstd in the PR).  Especially since we know that there's an issue with enabling concurrency in `zstd`. Having said that I'm not against removing this test. Also would like you're opinion on this [comment](https://github.com/open-telemetry/opentelemetry-collector/pull/9749#issuecomment-2032334943)